types/humidityMeasure.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "types/humidityMeasure",
+    "description": "Specifies a humidity measure including units (default is ).",
+    "type": "object",
+    "required": [],
+    "properties": {
+        "measurement": {
+            "type": "number",
+            "description": "Humidity measurement"
+        },
+        "unit": {
+            "type": "string",
+            "enum": [
+                "A93",
+                "PER",
```

Comment:
Line 1795 of [this](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Funece.org%2Fsites%2Fdefault%2Ffiles%2F2021-06%2Frec20_Rev17e-2021.xlsx&wdOrigin=BROWSELINK) says percentage should be P1. Do we want to follow that?
![image](https://user-images.githubusercontent.com/3358790/187140574-2d68bd89-2910-4f38-81cf-40b52cc1fae0.png)

```suggestion
                "P1",
```